.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Run QAWolf Tests
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  run-qa-wolf-tests:
+    runs-on: macos-latest
```

Comment:
This could update out from under us and possibly break things. Should we pin to `macos-14`? Also it's unclear to me from [these docs](https://github.com/actions/runner-images) if this x86 or arm. I guess it's arm which is good?